package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-find-rules",
-  "version": "4.1.0",
+  "version": "4.2.0",
```

Comment:
version numbers shouldn't be bumped in PRs.

Reply:
Sorry, I has fixed